templates/git-mirror/8/rancher-compose.yml

```diff
@@ -91,3 +97,5 @@ mirror:
     response_timeout: 3000
     healthy_threshold: 2
     unhealthy_threshold: 2
+logrotate:
+  start_on_create: false
```

Comment:
This doesn't seem to be honored - container is started on create. Any idea @cloudnautique ?